### PR TITLE
scry 0.9.1 (new formula)

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/scry.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  sha256 "53bf972557f8b6a697d2aa727df465d6e7d04f6426fcd4559a4d77c90becad81"
+  license "MIT"
+  head "https://github.com/crystal-lang-tools/scry.git"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "crystal" => :build
```
Comment (carlocab) [inline Formula/scry.rb:14]:
```suggestion
  depends_on "crystal" => :build
  depends_on "bdw-gc"
  depends_on "libevent"
  depends_on "pcre"
```

Comment (vladimyr) [reply Formula/scry.rb:9]: Wow, that was quite an _oversight_ 🤦

File: Formula/scry.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        #{request}
+      EOF
+
+      while (output = stdout.gets.split("Content-Length: ").first)
+        break if output.include? "\"id\":1"
+      end
+      assert_equal response, output
+    ensure
+      Process.kill "SIGKILL", wait_thr.pid
```
Comment (carlocab) [inline Formula/scry.rb:70]: Still fails the test. Probably from this line. You could drop this, if we expect the `scry` process to no longer be running at this point, or try `rescue Errno::ESRCH`.

**Review by carlocab — APPROVED** [review]
Thanks, @vladimyr.

**Pull request closed without merging**